db/src/lib.rs

```diff
@@ -69,7 +71,13 @@ fn u256_to_bytes(val: &U256) -> [u8; 32] {
 
 #[derive(sqlx::Type, Clone, Debug, Default)]
 #[sqlx(type_name = "U256", transparent)]
-struct PgU256(sqlx::types::BigDecimal);
+pub struct PgU256(sqlx::types::BigDecimal);
+
+impl PgU256 {
+    pub fn as_u64(&self) -> u64 {
+        todo!()
+    }
```

Comment:
fixme

Reply:
That's bad 😮‍💨. It means we lost a commit to the indexer during the split. Looking into it